### PR TITLE
pipeline-manager: demos with user defined functions

### PR DESCRIPTION
**Pull request by snkas**
If a `<filename>.sql` is found in a demos directory, also check for `<filename>.udf.rs` and `<filename>.udf.toml`. If present, these will be included in the demo as well.

Add a UDF tutorial demo in which to encode incoming binary data as base64.

### CODE REVIEW

**Comment by snkas** [comment]
@ryzhyk Either you could merge this into your PR, or after your PR is merged I can rebase it on main and merge separately.

**Review by ryzhyk — APPROVED** [review]

**Comment by ryzhyk** [comment]
> @ryzhyk Either you could merge this into your PR, or after your PR is merged I can rebase it on main and merge separately.

I think this is ready to merge, no need to combine it with my PR or wait for my PR to land.

File: scripts/start_manager.sh
Comment (lalithsuresh) [inline scripts/start_manager.sh:45]: you shouldn't need any of the rest either. This should do.

```
cargo run --bin=pipeline-manager --features pg-embed
```

Comment (ryzhyk) [reply scripts/start_manager.sh:45]: The rest is there so this script can be invoked with non-default settings. Not sure if anyone needs that...

**Pull request merged**
